### ISSUE TITLE
你好，关于论文中模型性能报告的问题

### ISSUE THREAD

**Issue by zyx1017** [YuehuaZhu/PAUP#2]
文中通过伯乐的框架进行实现，因为伯乐框架的集成性，我在上面同样了跑了SASRec 在数据集 ML-1m 的结果，与文中所报的结果差别十分大，甚至远好于PAUP的效果，其中NDCG@10 有0.16+的表现，recall@10有0.29+的表现，请问这是为什么？能分享一下你当时实现的相关参数吗？谢谢~

**Comment by YuehuaZhu** [comment]
你好，我们以[LightSANs](https://github.com/RUCAIBox/LightSANs)为baseline并提出改进，同时保持了一致的实验设定。此外，我们的对比方法结果和LightSANs文中报告是相近的，所以文中所报的对比结果与LightSANs也保持了一致。 由于对比方法较多，且不清楚你的实验方式，而本文以及LightSANs的参数设定均已开源，因此建议可以参考相关的设定进行多次对比。

**Comment by zyx1017** [comment]
@YuehuaZhu 大佬，对不起，我有个疑问：
在计算多头注意力得分时，为什么不需要像以前的做法一样，需要把之后的交互给mask掉？
比如以往的做法都会加上一个mask矩阵：attention_scores = attention_scores + attention_mask
而本代码见如下，并没有进行mask操作。对此感到疑问！希望能得到解答！！！
attention_scores = torch.matmul(query_layer, key_layer.transpose(-1, -2))
attention_scores = attention_scores / math.sqrt(self.attention_head_size)
        
\# normalize the attention scores to probabilities.
attention_probs = nn.Softmax(dim=-2)(attention_scores)
attention_probs = self.attn_dropout(attention_probs)
context_layer_item = torch.matmul(attention_probs, value_layer)
